external-api/api-tests/v1/eStore.api-test.ts

```diff
@@ -43,19 +43,37 @@ jest.mock('../../src/database/mongo-client', () => ({
   })),
 }));
 
-const mockExporterResponse = {
-  siteId: 'test',
-  status: 'Created',
+// Mock GovNotify email
+const mockSuccessfulEmailResponse = {
+  status: HttpStatusCode.Created,
+  data: {
+    content: {
+      body: 'Dear John Smith,\r\n\r\nThe status of your MIA for EuroStar has been updated.\r\n\r\nEmail: test@test.gov.uk\r\nPhone: +44 (0)202 123 4567\r\nOpening times: Monday to Friday, 9am to 5pm (excluding public holidays)',
+      from_email: 'test@notifications.service.gov.uk',
+      subject: 'Status update: EuroStar bridge',
+      unsubscribe_link: null,
+    },
+    id: 'efd12345-1234-5678-9012-ee123456789f',
+    reference: 'tmp1234-1234-5678-9012-abcd12345678-17133465334678',
+    scheduled_for: null,
+    template: {
+      id: 'tmp1234-1234-5678-9012-abcd12345678',
+      uri: 'https://api.notifications.service.gov.uk/services/abc12345-a123-4567-8901-123456789012/templates/tmp1234-1234-5678-9012-abcd12345678',
+      version: 24,
+    },
+    uri: 'https://api.notifications.service.gov.uk/v2/notifications/efd12345-1234-5678-9012-ee123456789f',
+  },
 };
 
-const mockApiResponse = {
-  status: HttpStatusCode.Ok,
+axiosMock.onPost(`${EXTERNAL_API_URL}/email`).reply(HttpStatusCode.Created, mockSuccessfulEmailResponse.data);
```

Comment:
Do we really need an entire example of the whole Notify response, could it for example just be the following to keep it simple?

```ts
const mockSuccessfulEmailResponse = {
  data: {
    content: {},
  },
};
```